admin_manual/ai/app_context_chat.rst

```diff
@@ -11,55 +11,56 @@ Context Chat is an :ref:`assistant<ai-app-assistant>` feature that is implemente
-   * This app makes use of the configured Text To Text Free prompt provider instead of running its own Language model, you will thus need only 4-8 cores for the embedding model
+      * 2 GB + additional 500MB for each request made to the backend if the Free text-to-text provider is not on the same machine
+      * 8 GB is recommended in the above case for the default settings
+   * This app makes use of the configured free text-to-text task processing provider instead of running its own language model by default, you will thus need 4+ cores for the embedding model only (backed configuration needs changes to make use of the extra cores, refer to :ref:`Configuration (Backend) <ai-app-context_chat-configuration-options-backend>`)
 
 * A dedicated machine is recommended
 
 Space usage
 ~~~~~~~~~~~
 
-This app employs a bundled Vector DB called `Chroma<https://github.com/chroma-core/chroma>`. All the users' textual data is duplicated, chunked and stored on disk in this vector DB along with semantic embedding vectors for the content.
-
-Assuming no shared files between users you can calculate with roughly the amount of textual data in user files (e.g. for PDFs, only the text counts, no images are kept). Any shared files will be duplicated per user, however, so, assuming all files are shared with all users you need to calculate with "the amount of textual data in user files * number of users". The reality will lie between these two estimates, of course.
+This app employs a bundled DB with Vector support called `PostgreSQL <https://www.postgresql.org/>`_. All the users' textual data is duplicated, chunked and stored on disk in this vector DB along with semantic embedding vectors for the content.
 
 Installation
 ------------
 
-0. Make sure the :ref:`Nextcloud Assistant app<ai-app-assistant>` is installed
-1. :ref:`Install AppAPI and setup a Deploy Demon<ai-app_api>`
-2. Install the *context_chat_backend* ExApp via the "External Apps" admin page in Nextcloud, or by executing
+1. Make sure the :ref:`Nextcloud Assistant app<ai-app-assistant>` is installed
+2. :ref:`Setup a Deploy Demon<ai-app_api> in AppAPI Admin settings`
+3. Install the *context_chat_backend* ExApp via the "External Apps" admin page in Nextcloud, or by executing (checkout the readme at https://github.com/nextcloud/context_chat_backend for manual install steps)
 
 .. code-block::
 
    occ app_api:app:register context_chat_backend
 
-3. Install the *context_chat* app via the "Apps" page in Nextcloud, or by executing
+4. Install the *context_chat* app via the "Apps" page in Nextcloud, or by executing
 
 .. code-block::
 
    occ app:enable context_chat
 
-4. Install a text generation backend like *llm2* (via the "External Apps" page in Nextcloud) or *integration_openai* (via the "Apps" page in Nextcloud), or by executing
+5. Install a text generation backend like *llm2* (via the "External Apps" page in Nextcloud) or *integration_openai* (via the "Apps" page in Nextcloud), or by executing
```

Comment:
Here I also think we should just link to the respective docs pages to avoid missing steps and duplicating information